llvm/docs/ReleaseNotes.md

```diff
@@ -355,6 +355,8 @@ Changes to LLDB
 
 * A new setting `target.launch-working-dir` can be used to set a persistent cwd that is used by default by `process launch` and `run`.
 
+* LLDB now creates modules in parallel, resulting in an average 2x speedup when launching and attaching.
```

Comment:
Good idea to highlight it.

Outside of us, I don't think most people know what "creates modules" is.  Maybe "reads binaries".  More accurately "scans binary symbol tables", but that's pushing what people would understand again.

I do have a little concern that we've only done this for the attach use-case on Darwin, and for the "load all dependent libraries" use case on all platforms.  Someone might attach to a process on linux and see no benefit at all, and be sad.

Reply:
So is this does load in parallel everywhere but you only measured it on Darwin, or has the possibility to do so everywhere but only does so right now on Darwin?

The former I would write as:
* LLDB can <general feature description>. On Darwin this has resulted in...

Implying that YMMV anywhere else.

The latter:
* LLDB running on Darwin will now load libraries in parallel when attaching or.... This gives an average 2x speedup.